website/docs/hail-format.md

```diff
@@ -306,4 +313,103 @@ We also provide a function to annotate the overall sumstats MatrixTable with the
\ No newline at end of file
+If your analysis requires the simultaneous analysis of summary statistics from multiple populations (and not the meta-analysis), you can load the data with a similar structure to the meta-analysis MatrixTable (one column per phenotype, with population information packed into an array of entries and columns) using `load_final_sumstats_mt(separate_columns_by_pop=False)`.
+
+## LD matrices
+
+The LD matrices are in [`BlockMatrix`](https://hail.is/docs/0.2/linalg/hail.linalg.BlockMatrix.html) format. Please refer to [Hail's documentation](https://hail.is/docs/0.2/linalg/hail.linalg.BlockMatrix.html) for available operations on `BlockMatrix`.
+```
+from hail.linalg import BlockMatrix
+bm = BlockMatrix.read(get_ld_matrix_path(pop='AFR'))
+```
+We note that the LD matrices were sparsified to a upper triangle (all elements of the lower triangle were zeroed out using [`BlockMatrix.sparsify_triangle`](https://hail.is/docs/0.2/linalg/hail.linalg.BlockMatrix.html#hail.linalg.BlockMatrix.sparsify_triangle)).
+
+### Variant indices
+
+To determine which row/column corresponds to which variant, we provide variant indices for `BlockMatrix` in `HailTable` format.
+```
+ht_idx = hl.read_table(get_ld_variant_index_path(pop='AFR'))
+```
+
+The variant indices table has the following schema and `idx` corresponds to a row/column index in `BlockMatrix`.
+```
+----------------------------------------
+Global fields:
+    'n_samples': int32
+    'pop': str
+----------------------------------------
+Row fields:
+    'locus': locus<GRCh37>
+    'alleles': array<str>
+    'idx': int64
+----------------------------------------
+Key: ['locus', 'alleles']
+----------------------------------------
+```
+
+### Extracting a subset of LD matrix
+
+To extract a subset of LD matrix, you first need to identify indices of your variants of interest. Here, we provide two examples:
+
+```
+# filter by interval
+interval = hl.parse_locus_interval('1:51572000-52857000')
+ht_idx = ht_idx.filter(interval.contains(ht_idx.locus))
+
+# or filter by a list of variant IDs (e.g., 1:51572412:A:G)
+ht = hl.import_table('/path/to/your/list')
+ht = ht.transmute(**hl.parse_variant(ht.variant)).key_by('locus', 'alleles')
+ht_idx = ht_idx.join(ht, 'inner')
+```
+
+Then, you can filter the LD matrix into a subset using [`BlockMatrix.filter`](https://hail.is/docs/0.2/linalg/hail.linalg.BlockMatrix.html#hail.linalg.BlockMatrix.filter):
+```
+idx = ht_idx.idx.collect()
+bm = bm.filter(idx, idx)
+```
+
+### Exporting a LD matrix to a flat file
+
+Finally, to export a LD matrix to a flat file (txt file), you can use [`BlockMatrix.export`](https://hail.is/docs/0.2/linalg/hail.linalg.BlockMatrix.html#hail.linalg.BlockMatrix.export):
+
+```
+# Note: when you apply any operation on BlockMatrix,
+# you need to write it to storage before export
+bm = bm.write('/path/to/tmp/bm', force_row_major=True)
```

Comment:
need to read after the write - switch to checkpoint?

Reply:
good catch! Actually, you don't need to read it back since `BlockMatrix.export` takes a bm path as input. Fixed here and next line.